src/openrct2/windows/footpath_scenery.c

```diff
@@ -0,0 +1,482 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
+/*****************************************************************************
+ * OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+ *
+ * OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+ * For more information, visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * A full copy of the GNU General Public License can be found in licence.txt
+ *****************************************************************************/
+#pragma endregion
+
+#include "../audio/audio.h"
+#include "../cheats.h"
+#include "../game.h"
+#include "../input.h"
+#include "../interface/themes.h"
+#include "../interface/viewport.h"
+#include "../interface/widget.h"
+#include "../localisation/localisation.h"
+#include "../rct2.h"
+#include "../sprites.h"
+#include "../world/footpath.h"
+#include "../world/scenery.h"
+#include "dropdown.h"
+
+#pragma region MEASUREMENTS
+
+#define WW						300
+#define WH						240
+#define PADDING					5
+#define IMGBUTTON_SIZE			23
+#define WIDGET_WIDTH			(WW - PADDING)
+#define LIST_WIDTH				(WW - 2 * PADDING - IMGBUTTON_SIZE)
+#define LIST_BUTTON_X			(LIST_WIDTH + PADDING)
+#define BUTTON_HEIGHT			16
+#define APPEND_BUTTON_LEFT		50
+#define APPEND_BUTTON_RIGHT		(WW - APPEND_BUTTON_LEFT)
+#define BUTTON_RIGHT			(PADDING + 250)
+#define WIDGET_HEIGHT			10
+
+#define DROPDOWN_RIGHT			(PADDING + 150)
+
+#define YPL(ROW)	((sint16)(PADDING + (BUTTON_HEIGHT * ROW)))
+#define HPL(ROW)	((sint16)(YPL(ROW) + BUTTON_HEIGHT))
+#define OHPL(ROW)	((sint16)(YPL(ROW) + WIDGET_HEIGHT))
+#define FHPL(ROW)	((sint16)(YPL(ROW) + IMGBUTTON_SIZE))
+
+#pragma endregion
+
+enum WINDOW_FOOTPATH_SCENERY_WIDGET_IDX
+{
+	WIDX_BACKGROUND,
+	WIDX_TITLE,
+	WIDX_CLOSE,
+
+	WIDX_RULE_LIST,
+
+	WIDX_CLEAR_RULES,
+
+	WIDX_OBJECT,
+	WIDX_OBJECT_DROPDOWN,
+
+	WIDX_ADD,
+};
+
+static rct_widget window_footpath_scenery_widgets[] = {
+	{ WWT_FRAME,			0,	0,								WW - 1,					0,				WH - 1,			0xFFFFFFFF, 						STR_NONE },
+	{ WWT_CAPTION,			0,	1,								WW - 2,					1,				14,				STR_FOOTPATH_SCENERY_TITLE,			STR_WINDOW_TITLE_TIP },
+	{ WWT_CLOSEBOX,			0,	WW - 13,						WW - 3,					2,				13,				STR_CLOSE_X,						STR_CLOSE_WINDOW_TIP },
+
+	{ WWT_SCROLL,			1,	PADDING,						LIST_WIDTH,				YPL(1),			OHPL(9),		SCROLL_VERTICAL,					STR_NONE },
```

Comment:
`LIGHT_BLUE` appears to be used for option dialogs. I think either tan or that pine green colour make sense: 
![image](https://cloud.githubusercontent.com/assets/737603/21681180/63273914-d34e-11e6-8baf-7354b6cbf140.png).